minifi-commons/minifi-commons-schema/src/main/java/org/apache/nifi/minifi/commons/schema/ConnectionSchema.java

```diff
@@ -60,30 +53,11 @@ public ConnectionSchema(Map map) {
         super(map, CONNECTIONS_KEY);
 
         sourceId = getOptionalKeyAsType(map, SOURCE_ID_KEY, String.class, CONNECTIONS_KEY, "");
```

Comment:
I believe this should now be required instead of optional (since there is no id vs name confusion)